script/import_scripts/socialcast/test/test_data.rb

```diff
@@ -2531,12 +2531,12 @@
                         {
                             "id": 108,
                             "url": "https://demo.socialcast.com/attachments/108",
-                            "filename": "108-reverse-a-algorithm-refernece-V7.pdf",
-                            "public_filename": "https://socialcast-demo.s3.amazonaws.com/tenants/5/attachments/108/108-reverse-a-algorithm-refernece-V7.pdf?AWSAccessKeyId=AKIAIV34WIEKJKCLRBBQ&Expires=1465128000&Signature=JP4%2BXB76kk%2BhCcuJQpZDSDx3NU4%3D",
+                            "filename": "108-reverse-a-algorithm-reference-V7.pdf",
+                            "public_filename": "https://socialcast-demo.s3.amazonaws.com/tenants/5/attachments/108/108-reverse-a-algorithm-reference-V7.pdf?AWSAccessKeyId=AKIAIV34WIEKJKCLRBBQ&Expires=1465128000&Signature=JP4%2BXB76kk%2BhCcuJQpZDSDx3NU4%3D",
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617142146
This url doesn't actually work

https://github.com/discourse/discourse/pull/12779#discussion_r617142185
@riking https://www.vmware.com/products/socialcast.html

https://github.com/discourse/discourse/pull/12779#discussion_r618873451
I'm not sure what to do about such things.

One approach is to suggest that someone else clean them up in some appropriate manner.

It's quite likely that shouldn't be me, since I can't realistically know how to replace such a thing, and also because even split this PR is still huge.

I can leave the fix in place, since it doesn't matter, or drop it.